code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -460,8 +460,8 @@
 			log_game("Warning: possible href exploit by [key_name(usr)] - attempted control of a mecha without can_dominate_mechs or a control beacon in the mech.")
 			return
 
-		if(controlled_mech)
-			to_chat(src, "<span class='warning'>You are already loaded into an onboard computer!</span>")
+		if(controlled_equipment)
+			to_chat(src, "<span class='warning'>You are already loaded into external equipment!</span>")
```

Comment:
was this intended to be [external_equipment]?

Reply:
what?

Reply:
er, I meant to say [controlled_equipment]